src/lib/rule-finder.js

```diff
@@ -1,11 +1,16 @@
 const path = require('path');
 
 const { ESLint, Linter } = require('eslint');
+const { builtinRules } = require('eslint/use-at-your-own-risk');
+const eslintPkg = require('eslint/package.json');
 const glob = require('glob');
+const semver = require('semver');
 const difference = require('./array-diff');
 const getSortedRules = require('./sort-rules');
 const normalizePluginName = require('./normalize-plugin-name');
 
+const isV8Eslint = semver.satisfies(eslintPkg.version, '< 9');
```

Comment:
eslint 8 supports flat config as well, it just doesn't require it.

Reply:
thank you, I has fixed